priv/templates/phx.gen.auth/settings_live.ex

```diff
@@ -0,0 +1,152 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>SettingsLive do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+
+  on_mount {<%= inspect auth_module %>, :mount_current_<%= schema.singular %>}
+
+  def render(assigns) do
+    ~H"""
+    <h1>Settings</h1>
+
+    <h3>Change email</h3>
+
+    <.form
+      id="email_form"
+      :let={f}
+      for={@email_changeset}
+      phx-submit="update_email"
+      phx-change="validate_email"
+    >
+      <%%= if @email_changeset.action == :insert do %>
+        <div class="alert alert-danger">
+          <p>Oops, something went wrong! Please check the errors below.</p>
+        </div>
+      <%% end %>
+
+      <%%= label f, :email %>
+      <%%= email_input f, :email, required: true, value: input_value(f, :email) %>
+      <%%= error_tag f, :email %>
+
+      <%%= label f, :current_password, for: "current_password_for_email" %>
+      <%%= password_input f, :current_password, required: true, name: "current_password", id: "current_password_for_email", value: input_value(f, :current_password) %>
+      <%%= error_tag f, :current_password %>
+
+      <div>
+        <%%= submit "Change email" %>
+      </div>
+    </.form>
+
+    <h3>Change password</h3>
+
+    <.form
+      id="password_form"
+      :let={f}
+      for={@password_changeset}
+      action={Routes.<%= schema.route_helper %>_session_path(@socket, :create, %{_action: "password_updated"})}
+      method="post"
+      phx-change="validate_password"
```

Comment:
It looks like I can update my password using the exact value of my current password. Is that something we should prevent?